docs/usage/tips-widget-creation.md

```diff
@@ -0,0 +1,114 @@
+# Pro-tips on widget creation
+
+Previously, the creation of widgets has been covered. To help users being more confident in creating widgets, here are some details to master the widget creation.
+
+<a id="howto-section"></a>
+## How to choose the appropriate widget visualization for your use case?
+
+We can classify the widgets in 3 different types.
+
+### Single dimension widgets
+
+Use these widgets when you would like to display information about one single type of object (entity or relation).
+
+- **Widget visualizations: number, list, list (distribution), timeline, donuts, radar, map, bookmark, tree map.**
+- Use case example: view the amount of malware in platform (number widget), view the top 10 threat actor group target a specific country (distribution list widget), etc.
+
+### Multi dimension widgets
+
+Use these widgets if you would like to compare or have some insights about similar types of object (up to 5).
+
+- **Widget visualizations: line, area, heatmap, vertical bars.**
+- Use case example: view the amount of malware, intrusion sets, threat actor groups added in the course of last month in the platform (line or area widget).
+
+!!! warning "Type of object in widget"
+
+  These widgets need to use the same "type" of object to work properly. You always need to add relationships in the filter view if you have selected a "knowledge graph" perspective. If you have selected the knowledge graph entity, adding "Entities" (click on `+ entities`) will not work, since you are not counting the same things.
+
+### Break down widgets
+
+Use this widget if you want to divide your data set into smaller parts to make it clearer and more useful for analysis.
+
+- **Widget visualization: horizontal bars.**
+- Use case example: view the list of malware targeting a country breakdown by the type of malware.
+
+![breakdown example](assets/Widget - breakdownExample.png)
+
+## Adding datasets to your widget
+
+Adding datasets can serve two purposes: comparing data or breakdown a view to have deeper understanding on what a specific dataset is composed of. 
+
+### Use Case 1: compare several datasets
+
+As mentioned in [How to choose the appropriate widget visualization for your use case?](tips_widget_creation.md#howto-section) section you can add data sets to compare different data. Make sure to add the same type of objects (entities or relations) to be able to compare the same objects, by using access buttons like `+`, `+ Relationships`, or `+ Entities`.
```

Comment:
I believe it will work better this way:
[How to choose the appropriate widget visualization for your use case?](#how-to-choose-the-appropriate-widget-visualization-for-your-use-case)
We can speak about it and I can show you the result on my local docs server

Reply:
I've used (#howto-section) and it works locally